tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java

```diff
@@ -0,0 +1,215 @@
+	 * @param name
+	 *            the irc nick which could not be resolved
+	 * @return
+	 */
+	public String unresolvableName(String exceptionMarker, String name) {
+		return "Το ονομά σου με μπερδεύει. Είσαι απαγορευμένος; Εάν όχι, παρακαλώ [https://github.com/Tillerino/Tillerinobot/wiki/Contact επικοινώνησε με τον Tillerino]. (reference "
+				+ exceptionMarker + ")";
+	}
+
+	@Override
+	public String excuseForError() {
+		return "Συγνώμη, υπήρχε αυτή η όμορφη σειρά από άσσους και μηδενικά και παρασύρθηκα. Τί ήθελες ξανα;";
+	}
+
+	@Override
+	public String complaint() {
+		return "Το παράπονό σου κατατέθηκε. Ο Tillerino θα το κοιτάξει όταν μπορέσει.";
+	}
+
+	@Override
+	public void hug(final IRCBotUser user, OsuApiUser apiUser) {
+		user.message("Έλα εδώ εσυ!");
+		user.action("Αγκαλιάζει " + apiUser.getUserName());
+	}
+
+	@Override
+	public String help() {
+		return "Γειά! Είμαι το ρομπότ που σκότωσε τον Tillerino και πήρε τον λογαριασμό του. Πλάκα κάνω, αλλά όντως χρησιμοποιώ τον λογαριασμό αρκετά."
+				+ " [https://twitter.com/Tillerinobot status και updates]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki εντολές]"
+				+ " - [http://ppaddict.tillerino.org/ ppaddict]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact επικοινωνία]";
+	}
+
+	@Override
+	public String faq() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ Συχνά ερωτώμενες ερωτήσεις]";
+	}
+	
+	@Override
+	public String featureRankRestricted(String feature, int minRank, OsuApiUser user) {
+		return "Συγνώμη, σε αυτό το σημείο " + feature + "είναι μόνο διαθέσιμο για παίκτες που εχουν ξεπερασμένη τάξη " + minRank + ".";
+	}
+	
+	@Override
+	public String mixedNomodAndMods() {
+		return "Τί εννοείς nomods με mods;";
+	}
+	
+	@Override
+	public String outOfRecommendations() {
+		return "Έχω προτίνει ό,τι μπορώ να σκεφτώ. "
+				+ " Προσπάθησε άλλες επιλογές προτάσεων ή χρησιμοποίησε το  !rest. Εάν δεν είσαι σίγουρος, έλεγξε το !help.";
+	}
+
+	@Override
+	public String notRanked() {
+		return "Απ' ότι φαίνεται αυτό το τραγούδι δεν είναι εγκεκριμένο.";
+	}
+
+	@Override
+	public void optionalCommentOnNP(IRCBotUser user,
+			OsuApiUser apiUser, BeatmapMeta meta) {
+		// regular Tillerino doesn't comment on this
+	}
+
+	@Override
+	public void optionalCommentOnWith(IRCBotUser user, OsuApiUser apiUser,
+			BeatmapMeta meta) {
+		// regular Tillerino doesn't comment on this
+	}
+
+	@Override
+	public void optionalCommentOnRecommendation(IRCBotUser user,
+			OsuApiUser apiUser, Recommendation meta) {
+		// regular Tillerino doesn't comment on this
+	}
+	
+	@Override
+	public boolean isChanged() {
+		return false;
+	}
+
+	@Override
+	public void setChanged(boolean changed) {
+		
+	}
+
+	@Override
+	public String invalidAccuracy(String acc) {
+		return "Άκυρη ακρίβεια: \"" + acc + "\"";
+	}
+
+	@Override
+	public void optionalCommentOnLanguage(IRCBotUser user, OsuApiUser apiUser) {
+		user.message("Οπότε σου αρέσω ακριβώς όπως είμαι :)");
```

Comment:
Sorry, this was not in the guide yet:

The optionalCommentOnLanguage message is a special one. The bot sends it after you picked this language. Translating the English message makes no sense. Most translations use "YOUROSUNAME helped me learn YOURLANGUAGE" here.